### ISSUE TITLE
Accept extra test arguments

### ISSUE THREAD

**Issue by pledbrook** [mjhugo/grails-auto-test#1]
The `auto-test` command should honour any extra test arguments passed to it. For example:

```
grails> auto-test unit:
grails> auto-test --rerun unit:spock
```

In particular, the delay from running the integration tests all the time could get annoying.

**Comment by mjhugo** [comment]
thanks for the feedback, Peter!  fixed in version 1.0.1

**Comment by pledbrook** [comment]
Wow, that was quick. Thanks!
